buildbot/master/init.sls

```diff
@@ -38,6 +38,7 @@ deploy-{{ common.servo_home }}/buildbot/master:
     - context:
         common: {{ common }}
         buildbot_credentials: {{ pillar['buildbot']['credentials'] }}
+        wpt_sync_credentials: {{ pillar['wpt-sync'] }}
```

Comment:
I'd prefer if these credentials got moved under `buildbot:credentials` so this line is not needed. The test credential would be removed from the `.travis/test_pillars/wpt-sync.sls` file and moved into the `.travis/test_pillars/buildbot/master.sls`.

Reply:
It's also used by https://github.com/servo/saltfs/blob/master/upstream-wpt-webhook/files/config.json#L2 so it won't actually be removed.

Reply:
I didn't realize that, keeping the current setup makes sense to me then.